client/blocks/signup-form/index.jsx

```diff
@@ -965,6 +968,18 @@ class SignupForm extends Component {
 			);
 		}
 
+		if ( this.props.currentUser ) {
+			return (
+				<ContinueAsUser
+					redirectPath={ this.props.redirectToAfterLoginUrl }
+					onChangeAccount={ () => {
```

Comment:
Could we move this function to a separate method of the component?

Reply:
Done. Added a new handle function